cmake/find/fastops.cmake

```diff
@@ -1,4 +1,4 @@
-if(NOT ARCH_ARM AND NOT OS_FREEBSD AND NOT OS_DARWIN)
+if(NOT ARCH_ARM AND NOT OS_FREEBSD AND NOT OS_DARWIN AND NOT ARCH_PPC64LE)
```

Comment:
Fastops is AVX math library, let's replace

NOT ARCH_ARM AND NOT ARCH_PPC64LE to ARCH_AMD64.